### ISSUE TITLE
Regarding profile picture

### ISSUE THREAD

**Issue by ChiragSinghal2001** [Harshdev625/SDEverse#203]
I think it should have a direct photo upload option so user can directly upload their photos. 

<img width="691" height="382" alt="Image" src="https://github.com/user-attachments/assets/a0dc7346-b9fa-4df2-b5a8-cf1d7fd9626c" />

If you find it relevant. Assign this feature to me @Harshdev625

**Comment by Harshdev625** [comment]
@ChiragSinghal2001 Any updates on this issue? Just checking in to see the progress.

**Comment by Harshdev625** [comment]
@JAINAM576 I’ve assigned this issue to you. You can start working on it. Please let me know how much time you’ll need and what changes you plan to make.

**Comment by JAINAM576** [comment]
@Harshdev625 
i will open pr today and for the changes i am thinking to add

2 options during edit profile

1. upload / change profile pic – it can be image / gif < 7mb
2. by avatar url

handle this using state and any one of them can be added.

-> if user uploads the image/gif -> check if it exceeds 7MB, show warning if it does -> converts to base64 format -> submit -> send to the backend -> upload to Cloudinary folder `profile_avatars` -> get the Cloudinary url -> save to the db -> acknowledgement -> show preview of uploaded image/GIF.

-> if url -> no need to process, just validation -> submit -> send to the backend -> store in db -> ack -> show preview of the URL.
